template.tpl

```diff
@@ -810,11 +810,24 @@ ___TEMPLATE_PARAMETERS___
                         "help": "By default, initial direct visits are tagged with the \"EMPTY\" campaign value. You can add a different initial empty value into this field. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#advanced-configuration-for-tracking-marketing-attribution\" \u003eRead more\u003c/a\u003e."
                       },
                       {
-                        "type": "TEXT",
-                        "name": "attributionExcludeReferrers",
-                        "displayName": "Exclude Referrers",
-                        "simpleValueType": true,
-                        "help": "Pass a comma-separated list of referring domains you want to exclude from campaign attribution. \u003ca href\u003d\"\"\u003eRead more\u003c/a\u003e."
+                        "type": "GROUP",
+                        "name": "attributionExcludeReferrersGroup",
+                        "displayName": "",
+                        "groupStyle": "NO_ZIPPY",
+                        "subParams": [
+                          {
+                            "type": "TEXT",
+                            "name": "attributionExcludeReferrersText",
```

Comment:
You're changing the name of the field. When the customer updates the GTM plugin, they'll lose the historic value for this field?

Reply:
Good call out. I updated to keep the name unchanged for backward compatibility.